### PR TITLE
docs(): Expanded release related documentation

### PR DESCRIPTION
**Pull request by kumar303**

### CODE REVIEW

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -88,3 +88,13 @@ When fixing up a pull request based on code review comments,
...
+
+## Creating a release
+
+To release a new version of `web-ext`, follow these steps:
+
+* Pull from master to make sure you're up to date.
+* Bump the version in `package.json`.
+* Commit and push the version change.
+* Tag master (example: `git tag 0.0.1`) and run `git push --tags upstream`.
+* Run `npm publish`.
```
Comment (andymckay) [inline CONTRIBUTING.md]: Can we do that automatically from travis?

**Comment by andymckay** [comment]
r+

File: CONTRIBUTING.md (hunk 1 above)
Comment (kumar303) [inline CONTRIBUTING.md]: Sure. I guess the advantage is anyone can push a tag and have it released?

Comment (andymckay) [inline CONTRIBUTING.md]: Yeah, that's really useful so you don't need to worry about npm credentials if someone is on vacation etc.

**Pull request merged**